libraries/exporters/src/main/java/org/eclipse/sw360/exporter/helper/ReleaseHelper.java

```diff
@@ -211,15 +214,30 @@ private void addFieldValueToRow(List<String> row, Release._Fields field, Release
                 addReleaseIdToRelationShipToRow(release.getReleaseIdToRelationship(), row);
                 break;
             case ATTACHMENTS:
-                String size = Integer.toString(release.isSetAttachments() ? release.getAttachments().size() : 0);
-                row.add(size);
+                if(isForProjectExport) {
+                	addSourceAttachmentsInformationToRow(release, row);
+                }else {
+                    String size = Integer.toString(release.isSetAttachments() ? release.getAttachments().size() : 0);
+                    row.add(size);
+                }
                 break;
             default:
                 Object fieldValue = release.getFieldValue(field);
                 row.add(fieldValueAsString(fieldValue));
         }
+    }    
+    
+    private void addSourceAttachmentsInformationToRow(Release release, List<String> row) throws SW360Exception{
+        Set<Attachment> attachments = CommonUtils.nullToEmptySet(release.getAttachments());
+        final Predicate<Attachment> isSourceAttachment = attachment -> attachment.getAttachmentType().name().equals("SOURCE") || attachment.getAttachmentType().name().equals("SOURCE_SELF");
```

Comment:
It is best practice to keep the constant / enum at first whenever you are comparing it with a variable.
Because there is a possibility of `getAttachmentType` returning null, and above code will throw `NullPointerException` in such cases.

Please refer: https://mkyong.com/java-best-practices/constant-value-should-always-come-first-in-comparison/

I suggest to change it something like this: `AttachmentType.SOURCE.equals(attachment.getAttachmentType())`